flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/ExecutionGraphCheckpointCoordinatorTest.java

```diff
@@ -68,6 +68,8 @@ public void testShutdownCheckpointCoordinatorOnFailure() throws Exception {
 
         graph.failJob(new Exception("Test Exception"), System.currentTimeMillis());
 
+        scheduler.closeAsync().get();
```

Comment:
I think these tests are no longer correct. The test name says that the CC should be shut down on an `ExecutionGraph` failure. However, this is no longer true. It's the same for the other tests.

Reply:
I've renamed the test to `DefaultSchedulerCheckpointCoordinatorTest` and renamed tests so they reflect current behavior. Also I've added test for auto suspending of the running job.